sensor/extrinsic_reflector_based_calibrator/include/extrinsic_reflector_based_calibrator/extrinsic_reflector_based_calibrator.hpp

```diff
@@ -227,9 +250,16 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
   // Tracking
   bool tracking_active_;
   int current_new_tracks_;
+  bool deletion_active_;
   TrackFactory::Ptr factory_ptr_;
   std::vector<Track> active_tracks_;
   std::vector<Track> converged_tracks_;
+
+  // Metrics
+  float output_cv_distance_error{0};
```

Comment:
Please pay attention at the way member variables are names.
Other codebases use a "m_" preffix, but at least in autoware, the "_" suffix is the standard

Reply:
Thanks. This variable is removed now.  `std::vector<float> output_metrics_;` is used now in commit https://github.com/tier4/CalibrationTools/pull/145/commits/d2299288a6f82602bcc8eead60a504072958ca67